### PR TITLE
feat: overview keyboard navigation 

### LINKED ISSUE #565: Keyboard navigation (2D) in slide overview
**Is your feature request related to a problem? Please describe.**
When in the overview menu, you can only select the scroll by clicking on it. Especially when having a large set of slides, this can get tedious easily, especially for devs who are used to use the keyboard 😛

**Describe the solution you'd like**
It would be nice to have the opportunity to select the current slide via keyboard in the overview section. You can focus slides via arrow keys (see image below). 

Pushing enter would close the overview and jump to the selected slide (though this can be programmed individually).

The whole feature is inspired by `reveal.js`.

The possible navigation via arrow keys

**Describe alternatives you've considered**
Not having this feature (as it is only for convenience).

### PR DESCRIPTION
**Pull request by WingLim**
Navigation action:
- [x] left
- [x] right
- [x] up
- [x] down
- [x] enter

Cypress test:
- [x] left
- [x] right
- [x] up
- [x] down
- [x] enter

Resolve #565

### CODE REVIEW

**Comment by antfu** [comment]
I can't get it working locally. Is it ready?

**Comment by WingLim** [comment]
All shortcut not working or just up an down?
I found a bug of the up and down, fixing now.

**Comment by WingLim** [comment]
@antfu I add some cypress test to make sure it is working for now.
Plz review it if you have free time.

File: cypress/integration/examples/basic.spec.ts [hunk 1]
```diff
@@ -72,7 +72,6 @@ context('Basic', () => {
     cy.get('body')
       .type('{UpArrow}')
       .url()
-      .should('eq', 'http://localhost:3030/6')
```
Comment (WingLim) [inline cypress/integration/examples/basic.spec.ts:75]: Oops, I made a mistake and accidentally deleted this line.

Comment (antfu) [reply cypress/integration/examples/basic.spec.ts:75]: You can do another PR to bring it back. Thanks

**Pull request merged**